oolab/src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -2,19 +2,20 @@
 
 import java.util.*;
 
-abstract class AbstractWorldMap implements IWorldMap, IPositionChangeObserver {
-
-    final Vector2d start_vector = new Vector2d(0,0);
+public abstract class AbstractWorldMap implements IWorldMap, IPositionChangeObserver {
     protected List<Animal> animalsList = new ArrayList<>();
     protected Map<Vector2d, Animal> animalsMap = new HashMap<>();
+    protected MapBoundary mapBoundary=new MapBoundary();
 
     @Override
     public boolean place(Animal animal) {
```

Comment:
skoro rzucamy wyjątkiem to nie zwrócimy tu nigdy false -> ta metoda nie powinna już zwracać boolean